resources/assets/components/InboxItem/index.js

```diff
@@ -11,6 +11,14 @@ class InboxItem extends React.Component {
     }
   }
 
+  calculateAge(date) {
```

Comment:
we have a [`helpers.js`](https://github.com/DoSomething/rogue/blob/master/resources/assets/helpers.js)